DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Kévin Gomez <contact@kevingomez.fr>
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * @var bool
+     */
+    private $debug;
+
+    /**
+     * @param bool $debug
+     */
+    public function  __construct($debug)
+    {
+        $this->debug = (bool) $debug;
+    }
+
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('hoathis_ruler');
```

Comment:
It's not the same usage. The previous one is used by the framework to avoid collisions between data-dollectors whereas this one will be used by the users to configure the bundle (it's the main alias under which the configuration will be defined).